Lesson14/script.js

```diff
@@ -1,12 +1,22 @@
 // 9. Declare a variable number6 and initialize it to 6. Declare a variable number7 and initialize it to `++number6` (number6 incremented using prexif ++ operator). Log both values to the console.
-
+let number6 = 6;
+let number7 = ++number6;
```

Comment:
Since value of `number7` never changes, it's better to create it as a constant:
```suggestion
const number7 = ++number6;
```